src/Main.java

```diff
@@ -1,5 +1,129 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+
+
+        int age = 18;
+        if (age >= 18) {
+            System.out.println("Если " + age + " можно водить");
+        }
+        if (age < 18) {
+            System.out.println(" Если возраст " + age + " не можешь водить ");
+        }
+
+
+        int temperature = 4;
+        if (temperature < 5) {
+            System.out.println("На улице" + temperature + " градусов,нужно надеть шапку");
+        } else {
+            System.out.println("На улице" + temperature + " градусов,можно идти без шапки");
+        }
+
+
+        int speed = 81;
+        if (speed > 60) {
+            System.out.println(" Если скорость" + speed + "придется платить штраф");
+        } else {
+            System.out.println("Если скорость" + speed + "штрафа не будет");
+        }
+
+
```

Comment:
```suggestion
        age = 12;
```
возраст можно было переприсвоить, если что